aspnet/getting-started/choosing-the-right-dotnet.rst

```diff
@@ -1,37 +1,65 @@
 Choosing the Right .NET For You on the Server
 =============================================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
-ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. When selecting a DNX to use you also have a choice of .NET flavors to pick from: .NET Framework (CLR), :doc:`.NET Core </conceptual-overview/dotnetcore>` (CoreCLR) or `Mono <http://mono-project.com>`_. Which .NET flavor should you choose? Let's look at the pros and cons of each one.
+ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. DNX come in three .NET versions: 
+
+- .NET Framework (Also known as the "*Full*" .NET Framework)
```

Comment:
So do you want it .NET Framework (Also known as the "*full*" .NET Framework) 
or do you want me to remove  (Also known as the "*full*" .NET Framework) 
There is a ginormous ambiguity in the name .NET Framework, and full disambiguate it. No so for you and the team, but anyone new to .Net is confused. Even MS blogs call it the full .net framework. 

You can see in there how depending on the application type you’d want to create (Client apps or Web applications and Services) you now have two choices:

- You can use the full .NET Framework that’s evolving and now is branded as .NET Framework 4.6

- You can use the smaller and scenario-specialized new framework called .NET Core 5.

A different point of view, more “framework-based” of the same technologies is this othe
see http://blogs.msdn.com/b/cesardelatorre/archive/2014/11/18/what-is-net-core-5-and-asp-net-5-within-net-2015-preview.aspx